.travis.yml

```diff
@@ -43,8 +50,28 @@ matrix:
     - rvm: 1.8.7
       gemfile: gemfiles/mime_types_2.3.gemfile
     - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_2.4.gemfile
+    - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_2.5.gemfile
+    - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_2.6.gemfile
+    - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_2.6_columnar.gemfile
+    - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_2.99.gemfile
+    - rvm: 1.8.7
       gemfile: gemfiles/mime_types_2.latest.gemfile
     - rvm: 1.8.7
+      gemfile: gemfiles/mime_types_3.0.gemfile
+    - rvm: 1.9.2
+      gemfile: gemfiles/mime_types_3.0.gemfile
+    - rvm: 1.9.3
+      gemfile: gemfiles/mime_types_3.0.gemfile
```

Comment:
Maybe this? 

```yaml
matrix:
  include:
  - rvm: jruby-9.0.3.0
```

cargo culted from https://github.com/bundler/gemstash/pull/47/files#diff-354f30a63fb0907d4ad57269548329e3R5

though maybe set the JRUBY_OPTS in travis as

```yaml
  - rvm: jruby-9.0.3.0
    env: JRUBY_OPTS="$JRUBY_OPTS -J-Xmx1536m -J-XX:MaxPermSize=192m"
```

or as in  https://github.com/rspec/rspec-dev/pull/115

```yaml
  - rvm: jruby-9.0.3.0
    env: JRUBY_OPTS='--server -Xcompile.invokedynamic=false'
```
```yaml
    - rvm: jruby-19mode
      env: JRUBY_OPTS="$JRUBY_OPTS --debug"  # for simplecov
    - rvm: jruby-20mode
      env: JRUBY_OPTS="$JRUBY_OPTS --debug"  # for simplecov
    - rvm: jruby-head
      env: JRUBY_OPTS="$JRUBY_OPTS --debug"  # for simplecov
```

like in https://github.com/dkubb/sql/blob/36bc3d3834ed36e500fa8184c6c9f57c6f8af6b0/.travis.yml

ref: https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables

`JRUBY_OPTS="--server -Dcext.enabled=false -Xcompile.invokedynamic=false"`